Fix the rule that generates the .moc file

The Linux kernel has a "cmd" make function, but we don't have it in
Buildroot, so we need to adjust this rule.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -307,11 +307,8 @@
 
 $(obj)/qconf.o: $(obj)/qconf.moc
 
-quiet_cmd_moc = MOC     $@
-      cmd_moc = $(KC_QT_MOC) -i $< -o $@
-
 $(obj)/%.moc: $(src)/%.h $(obj)/.tmp_qtcheck
-	$(call cmd,moc)
+	$(KC_QT_MOC) -i $< -o $@
 
 # Extract gconf menu items for I18N support
 $(obj)/gconf.glade.h: $(obj)/gconf.glade
